examples/eSealing/src/utils.ts

```diff
@@ -8,23 +18,46 @@ import { E_SEALING_CONTRACT_NAME, E_SEALING_RAW_SCHEMA } from './constants';
  */
 export async function register(
     connection: WalletConnection,
+    grpcClient: ConcordiumGRPCClient,
     account: string,
     fileHashHex: string,
     index: bigint,
     subindex = 0n
 ) {
+    const contractAddress = ContractAddress.create(index, subindex);
+    const receiveName = ReceiveName.fromString(`${E_SEALING_CONTRACT_NAME}.registerFile`);
+
+    const schema = moduleSchemaFromBase64(E_SEALING_RAW_SCHEMA);
+    const serializedParameters = serializeUpdateContractParameters(
+        ContractName.fromString(E_SEALING_CONTRACT_NAME),
+        EntrypointName.fromString('registerFile'),
+        fileHashHex,
+        schema.value
+    );
+
+    const invokeResult = await grpcClient.invokeContract({
+        contract: contractAddress,
+        method: receiveName,
+        invoker: AccountAddress.fromBase58(account),
+        parameter: serializedParameters,
+    });
+
+    if (invokeResult.tag === 'failure') {
+        throw Error('Transaction would fail!');
```

Comment:
It would be helpful if that error message reports the reason for the failure. The following, I suggest to cover: 

- Return the `rejectReason` of the smart contract if present.
- Return the `invokeResult.tag` (if it is not `failure`) since that is where we can see `OutOfEnergy` errors:
   https://github.com/Concordium/concordium-smart-contract-tools/blob/main/front-end-tools/src/utils.ts#L209


```
    const fullEntryPointName = `${contractName.value}.${entryPoint.value}`;

    if (!res || res.tag === 'failure') {
        const rejectReason = JSON.stringify(
            ((res as InvokeContractFailedResult)?.reason as RejectedReceive)?.rejectReason
        );

        throw new Error(
            `RPC call 'invokeContract' on method '${fullEntryPointName}' of contract '${contractIndex}' failed.
            ${rejectReason !== undefined ? `Reject reason: ${rejectReason}` : ''}`
        );
    }
```

Reason for adding this reporting of errors now:
Before, people who reported problems while using this front end were able to give us a "failedTransactionHash" that we could use for further investigation. We don't have this anymore, so reporting potential problems here is what we have to rely on now.

This comment applies to the other examples as well.